src/commands/createFunctionApp.ts

```diff
@@ -4,16 +4,23 @@
  *--------------------------------------------------------------------------------------------*/
 
 import { AzureTreeDataProvider, IActionContext, IAzureNode, IAzureParentNode } from 'vscode-azureextensionui';
+import { FunctionAppTreeItem } from '../tree/FunctionAppTreeItem';
 import { nodeUtils } from '../utils/nodeUtils';
 
-export async function createFunctionApp(actionContext: IActionContext, tree: AzureTreeDataProvider, arg?: IAzureParentNode | string): Promise<string> {
+export async function createFunctionApp(actionContext: IActionContext, tree: AzureTreeDataProvider, subscription?: IAzureParentNode | string, resourceGroup?: IAzureParentNode<FunctionAppTreeItem> | string): Promise<string> {
```

Comment:
When will resourceGroup ever be `IAzureParentNode<FunctionAppTreeItem>`? Shouldn't it just be string?

Reply:
I wasn't sure how it was going to be used, but I thought a user could just pass another function app and use the same resource group as the one passed.